tools/rna_tools/rnaformer/infer_rnaformer.xml

```diff
@@ -0,0 +1,202 @@
+<tool id="infer_rnaformer" name="@EXECUTABLE" version="@VERSION@" profile="22.05">
+    <description>Predict the secondary structure of an RNA with RNAformer</description>
+    <macros>
+        <token name="@EXECUTABLE@">RNAformer</token>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements">
+        <requirement type="package" version="1.83">biopython</requirement>
+    </expand>
+    <expand macro="stdio" />
+    <expand macro="version_command" />
+    <command detect_errors="exit_code"><![CDATA[python '$script_file' > '$output'
+]]></command>
+<configfiles>
+        <configfile name="script_file"><![CDATA[import RNAformer
+import os
+import argparse
+import torch
+import urllib.request
+import logging
+from collections import defaultdict
+import torch.cuda
+import loralib as lora
+from RNAformer.model.RNAformer import RiboFormer
+from RNAformer.utils.configuration import Config
+
+from Bio import SeqIO
+
+config_file_path = '$__tool_directory__/test-data/RNAformer_32M_config_intra_family_finetuned.yml'
+model_file_path = '$__tool_directory__/test-data/RNAformer_32M_state_dict_intra_family_finetuned.pth'
```

Comment:
That file is too big, usually, we do not ship models with the tool. How often does this model change, gets updated?

Reply:
Never. It is only used for inference so would just need to be downloaded once if it doesn't exist which is what I previously did but thought it might be too large for the tool to download and was causing the tests to fail. I will revert it to just check if the model and configuration files exist and download it if not.